test/models/report_test.rb

```diff
@@ -4,17 +4,17 @@
 
 class ReportTest < ActiveSupport::TestCase
   test '#editable?' do
-    current_user = users(:Alice)
-    report1 = current_user.reports.new
-    assert report1.editable?(users(:Alice))
+    alice = users(:alice)
+    alice_report = alice.reports.new
+    assert alice_report.editable?(alice)
 
-    current_user = users(:Bob)
-    report2 = current_user.reports.new
-    assert_not report2.editable?(users(:Alice))
+    bob = users(:bob)
+    bob_report = bob.reports.new
+    assert_not bob_report.editable?(alice)
   end
 
   test '#created_on' do
-    report1 = users(:Alice).reports.new(created_at: '2021-10-14 02:10:11.678236 UTC')
-    assert_equal report1.created_on.to_s, '2021-10-14'
+    report = Report.new(created_at: '2021-10-14 02:10:11.678236 UTC')
+    assert_equal report.created_on.to_s, '2021-10-14'
```

Comment:
```suggestion
    assert_equal report.created_on, '2021-10-14'.to_date
```

のようにしてDate型同士の比較をした方が良いかもしれません。（一般論として、`to_s`すると文字列に変換したタイミングで何か大事な情報が失われる可能性もあるかもしれないため）